test/unit/annotation_spec.js

```diff
@@ -343,6 +411,154 @@ describe('annotation', function() {
 
       expect(markupAnnotation.creationDate).toEqual(null);
     });
+
+    it('should correctly parse T and not ' +
+        'parse IRT and RT when not defined', function (done) {
+      dict.set('Type', Name.get('Annot'));
+      dict.set('Subtype', Name.get('Text'));
+      dict.set('T', 'Author Name');
+
+      const xref = new XRefMock([
+        { ref, data: dict, },
+      ]);
+
+      AnnotationFactory.create(xref, ref, pdfManagerMock,
+        idFactoryMock).then(({ data, }) => {
+          expect(data.title).toEqual('Author Name');
+
+          expect(data.inReplyTo).toBeUndefined();
+          expect(data.replyType).toBeUndefined();
+          done();
+        }, done.fail);
+    });
+
+    it('should correctly parse IRT and set default RT.', function (done) {
+      const annotationDict = new Dict();
+      annotationDict.set('Type', Name.get('Annot'));
+      annotationDict.set('Subtype', Name.get('Text'));
+      const annotationRef = new Ref(819, 0);
+
+      const replyDict = new Dict();
+      replyDict.set('Type', Name.get('Annot'));
+      replyDict.set('Subtype', Name.get('Text'));
+      replyDict.set('IRT', annotationRef);
+
+      const replyRef = new Ref(820, 0);
+
+      const xref = new XRefMock([
+        { ref: annotationRef, data: annotationDict, },
+        { ref: replyRef, data: replyDict, }
+      ]);
+
+      AnnotationFactory.create(xref, replyRef, pdfManagerMock,
+        idFactoryMock).then(({ data, }) => {
+
+          expect(data.inReplyTo).toEqual(annotationRef.toString());
+          expect(data.replyType).toEqual('R');
+          done();
+        }, done.fail);
+    });
+
+    it('should correctly parse IRT, RT and take Contents ' +
```

Comment:
There are basically two situations when IRT/RT is parsed: either it's a group annotation (and its contents should be inherited) or it's a reply annotation. How about updating this test and the one below so that this is clear from the name and so all fields that can be changed, namely title, contents, creation date, modification date, `hasPopup` and color, are checked. That way it's clear which two scenarios we expect and we make sure that all fields that can be changed are in fact tested instead of only the `Contents` field. Both tests are already pretty close to this, but needs some tweaks to get there, mainly by changing the name and testing all fields.

This one can then be called `should parse IRT/RT for a group type` and the other one can then be called `should parse IRT/RT for a reply type`.